### PR TITLE
gitlabci webhook tutorial

### PR DESCRIPTION
**Pull request by yuanbohan**
@xtang @loddit

### CODE REVIEW

**Comment by xtang** [comment]
LGTM

**Comment by xtang** [comment]
cc @loddit

**Comment by xtang** [comment]
@loddit poke

File: robots/gitlabci.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+选择对应的项目，点击 「Add project to CI」按钮完成添加
+
+![](/images/tutorial/gitlabci_dashboard.png)
+
+## 进入 Project 页面
+添加 CI 完成之后进入项目设置页面， 按照「How to setup CI for this project」来正确配置 GitLab CI
+
+![](/images/tutorial/gitlabci_setting.png)
+
+## 设置 webhook
```
Comment (loddit) [inline robots/gitlabci.md]: Web hooks 和截图保持一致吧

**Comment by loddit** [comment]
LGTM

**Pull request merged**